derive/Cargo.toml

```diff
@@ -2,7 +2,9 @@
 name = "clickhouse-derive"
 version = "0.2.0"
 description = "A macro for deriving clickhouse::Row"
-authors = ["Paul Loyd <pavelko95@gmail.com>"]
+authors = ["ClickHouse Contributors <???>"] # REVIEW
+repository = "https://github.com/ClickHouse/clickhouse.rs"
+homepage = "https://clickhouse.com"
 edition = "2021"
 license = "MIT OR Apache-2.0"
```

Comment:
Let's keep a single license to avoid questions. The repo contains `MIT` file, let's keep the `MIT` license and avoid adding another one

Reply:
AFAIK the `license` field is more important than files, so currently the crate is already double-licensed.

And sometimes people asks about it https://github.com/loyd/rscam/issues/5

Reply:
So, I'm going to leave double license here